task/mypy-lint/0.1/mypy-lint.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: mypy-lint
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: linter
+    tekton.dev/displayName: "Mypy(Python) linter"
+spec:
+  description: >-
+    This task can be used to perform lint check on python files
```

Comment:
Can we have some better description like: A workspace containing files that we want to check.

As workspace may not necessarily contain fetched git repository it may also contain files mounted using ConfigMap